README.rst

```diff
@@ -0,0 +1 @@
+README.txt
```

Comment:
@tomgross wouldn't make more sense to just rename it? setup.py most probably should be updated then